cli/dcoscli/tables.py

```diff
@@ -209,6 +209,100 @@ def service_table(services):
     return tb
 
 
+def job_table(job_list):
+    """Returns a PrettyTable representation of the job list from Metronome.
+
+    :param job_list: jobs to render
+    :type job_list: [job]
+    :rtype: PrettyTable
+    """
+
+    fields = OrderedDict([
+        ('id', lambda s: s['id']),
+        ('Description', lambda s: _truncate_desc(s['description'])),
+        ('Status', lambda s: _job_status(s)),
+        ('Last Succesful Run', lambda s: s['history']['lastSuccessAt']),
+    ])
+    tb = table(fields, job_list, sortby="ID")
+    tb.align['ID'] = 'l'
+    tb.align["DESCRIPTION"] = 'l'
+    tb.align["STATUS"] = 'l'
+
+    return tb
+
+
+def job_history_table(schedule_list):
+    """Returns a PrettyTable representation of the job history from Metronome.
+
+    :param schedule_list: job schedule list to render
+    :type schedule_list: [history]
+    :rtype: PrettyTable
+    """
+
+    fields = OrderedDict([
+        ('id', lambda s: s['id']),
+        ('started', lambda s: s['createdAt']),
+        ('finished', lambda s: s['finishedAt']),
+    ])
+    tb = table(fields, schedule_list, sortby="STARTED")
+    tb.align['ID'] = 'l'
+
+    return tb
+
+
+def schedule_table(schedule_list):
+    """Returns a PrettyTable representation of the schedule list of a job from Metronome.
+
+    :param schedule_list: schedules to render
+    :type schedule_list: [schedule]
+    :rtype: PrettyTable
+    """
+
+    fields = OrderedDict([
+        ('id', lambda s: s['id']),
+        ('cron', lambda s: s['cron']),
+        ('enabled', lambda s: s['enabled']),
+        ('next run', lambda s: s['nextRunAt']),
+        ('concurrency policy', lambda s: s['concurrencyPolicy']),
+    ])
+    tb = table(fields, schedule_list)
+    tb.align['ID'] = 'l'
+    tb.align['CRON'] = 'l'
+
+    return tb
+
+
+def _truncate_desc(description, truncation_size=35):
```

Comment:
Why is the default truncation size so short? We use `80` in `package_search_table`